pol-core/ecompile/ECompileMain.cpp

```diff
@@ -417,6 +506,16 @@ void process_file_wrapper( const char* path )
     if ( !keep_building )
       throw;
   }
+
+  if ( watch_source )
```

Comment:
See comment at globals

Reply:
`watch_source` is read in two places: (1) `process_file_wrapper` to determine if `add_dependency_info` should be called, (2) end of `run()` to determine if `EnterWatchMode` should be called. I think`watch_source` should remain global.